src/main/java/com/neonlab/product/apis/DeleteProductApi.java

```diff
@@ -0,0 +1,33 @@
+package com.neonlab.product.apis;
+import com.neonlab.common.annotations.Loggable;
+import com.neonlab.common.dto.ApiOutput;
+import com.neonlab.common.expectations.InvalidInputException;
+import com.neonlab.common.utilities.StringUtil;
+import com.neonlab.product.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+
+@Service
+@Loggable
+public class DeleteProductApi {
+
+    @Autowired
+    private ProductService productService;
+
+    public ApiOutput<?>deleteProductApi(String code){
```

Comment:
Please refer to the whimsical comment [link](https://whimsical.com/validations-XtpgqbUqyAuUTKGfjhxHAd) for explaination. 

Please update the request to include the quantity as mandatory field

cc- @sjha24